### PR TITLE
Cherry pick from master

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Step 1 to prepare release for bridge runtime upgrade

Develop has the latest polkadot version. But develop branch is far behind with what we have in Master. This PR is to cherry pick all the commits from master to develop and after this is good and merged, we can rebase bridge_wip branch.

### CODE REVIEW

**Review by holygits — APPROVED** [review]
anything you'd like checked in particular? cherry-picks seems fine to me

**Comment by KarishmaBothara** [comment]
> anything you'd like checked in particular? cherry-picks seems fine to me

There is one last commit I would want to cherry pick (https://github.com/cennznet/api.js/pull/341). This is now in master..
Also we might want to fix the docker image to use node's develop branch(latest) https://hub.docker.com/r/cennznet/cennznet/tags?page=1&ordering=last_updated
Once we have latest one, then I can enable the existential deposit on asset info and run all the test..

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,9 +4,11 @@ on:
   pull_request:
     branches:
     - develop
+    - master
```
Comment (holygits) [inline .github/workflows/ci.yml:7]: can you remove `master` from here

Comment (KarishmaBothara) [reply .github/workflows/ci.yml]: done...

**Review by eupston — COMMENTED** [review]
Nice flying through CI so lgtm

**Pull request merged**